# FlexSensorData.msg
# Message for flex sensor readings

std_msgs/Header header
float32[4] resistance_values  # Resistance values in ohms for 4 flex sensors